ext/mbstring/tests/iso2022jp_encoding.phpt

```diff
@@ -52,8 +52,6 @@ function testValid($from, $to, $encoding, $bothWays = true) {
 			$from = substr($from, 3, strlen($from) - 3);
 		/* If the string switches to a different charset, it should switch back to
```

Comment:
Code was deleted here, but not the comment which explains the code. The comment should be deleted as well.